jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/WebAppGeneratorTest.java

```diff
@@ -172,7 +175,7 @@ void withOverriddenProperties_shouldAddImageConfiguration() throws IOException {
           .extracting(BuildConfiguration::getAssembly)
           .hasFieldOrPropertyWithValue("excludeFinalOutputArtifact", true)
           .hasFieldOrPropertyWithValue("user", "root")
-          .extracting("inline.files").asList().extracting("destName")
+          .extracting("inline.files").asInstanceOf(InstanceOfAssertFactories.list(Type.class)).extracting("testName")
```

Comment:
typo
```suggestion
          .extracting("inline.files").asInstanceOf(InstanceOfAssertFactories.list(Type.class)).extracting("destName")
```

Reply:
I thought it is a "TestName" I'll Change it